Remotion/Web/Core/Res/Themes/NovaViso/HTML/Common.css

```diff
@@ -471,13 +471,15 @@ img.disabled,
   filter: grayscale(100%);
 }
 
+.remotion-scrollable-theme,
```

Comment:
Not sure about the same I prefer `remotion-scrollable` as you explicitly don't want any other theming so the `-theme` suffix implies something else.